css-grid/css-grid_place-elements/css/styles.css

```diff
@@ -42,30 +42,32 @@ body {
 }
 .element-1 {
   background-color: #3a67a4;
-  /* Add styles here */
+  grid-column: 3/5;
 }
 
 .element-2 {
   background-color: #3a67a4;
-  /* Add styles here */
+  grid-column: 5/11;
```

Comment:
Just in case there are more columns are added in the future, I would use `grid-column: 5/-1;`

Reply:
Looks good!